javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,46 @@
+
+var width = 50;
+var height = 70;
+var area = width * height;
+
+console.log('value of area:', area);
+console.log('typeof area:', typeof 3500);
+
+var bill = 1500;
+var payment = 1600;
+var change = payment - bill;
+
+console.log('value of change:', change);
+console.log('typeof change:', typeof 100);
+
+var quizzes = 100;
+var midterm = 50;
+var final = 90;
+var grade = ((quizzes + midterm + final) / 3);
+
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof 80);
+
+var firstName = 'Henry';
+var lastName = 'Nichols';
+var fullName = firstName + ' ' + lastName;
+
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof 'Henry Nichols');
+
+var pH = 10;
+var isAcidic = (pH > 7);
+
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof true);
+
+var headCount = 30;
+var isSparta = headCount === 300;
+
+console.log('value of isSparta:', isSparta);
+console.log('typeof isSparta', typeof false);
+
+var motto = fullName + ' ' + 'is the GOAT';
+
+console.log('value of motto:', motto);
+console.log('typeof motto:', typeof 'Henry Nichols is the GOAT');
```

Comment:
Instead of typing the value directly into your console logs, you should be passing the variable that holds that value.